staubli_rx160_gazebo/launch/spawn_rx160.launch

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0"?>
+<launch>  
+
+  <!-- remap topics to conform to ROS-I specifications -->
+  <remap from="/arm_controller/follow_joint_trajectory" to="/joint_trajectory_action" />
+  <remap from="/arm_controller/state" to="/feedback_states" />
+  <remap from="/arm_controller/command" to="/joint_path_command"/>
+
+  <!-- send robot urdf to ROS param server -->
+  <include file="$(find staubli_rx160_gazebo)/launch/load_rx160.launch" />
+
+  <!-- push robot_description to factory and spawn robot in gazebo -->
+  <node name="spawn_gazebo_model" pkg="gazebo_ros" type="spawn_model" 
+        output="screen" respawn="false"
+        args="-urdf
+              -param robot_description
+              -model rx160
+              -x 0.0
+              -y 0.0
+              -z 0.0
+              -R 0.0
+              -P 0.0
+              -Y 0.0" />
+
+    <!-- convert joint states to TF transforms for rviz, etc -->
+  <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher"
+  output="screen">
+  </node>
+
+  <!-- init and start Gazebo ros_control interface -->
+  <include file="$(find staubli_rx160_gazebo)/launch/rx160_control.launch"/>
+  
+</launch>
```

Comment:
@gavanderhoorn The reason why there are two launch files to spawn two distinct robots is based on our previous discussion in ros-industrial/staubli#10. :-)

I recall I had parametrised something and you suggested it would be more intuitive to have separate files (despite redundancy); you convinced me there, so I adopted that idea.

Regarding the `_control` launch file: what I understand by spawning a robot in Gazebo is loading the necessary modules in order for such robot to be simulated, and for Gazebo to expose an interface (in this case, ros_control). Since this is a ROS package, controllers are an intrinsic component to the simulation. Does that make sense?